proprietary/design-tokens/src/imported/nl/utrecht-alert.json

```diff
@@ -142,5 +142,27 @@
         }
       }
     }
+  },
+  "lux": {
+    "alert": {
+      "heading": {
+        "font-family": {
+          "value": "{lux.font-family.primary}",
```

Comment:
Als het er altijd uit moet zien als een `heading-4`, moeten `font-family` en `font-weight` dan niet daarheen refereren?

Reply:
Mmh ik pas hier nu common/brand tokens toe, ook voor de font-size en line-height. 

Wat jij voorstelt kan ook, is alleen een ander aanpak dan ik nu uitvoer. De common-tokens voor de heading levels hebben dan niet echt een bestaansrecht. Zou ik over na moeten denken dan.

Reply:
Mijn voorstel zou zijn om het nu zo te laten en deze vraag even parkeren voor later.